### DISCUSSION TITLE
Opening the downloaded "The Radio Regulations 2024" causes lag.

### DISCUSSION THREAD

**Discussion by BI1XVY** [sumatrapdfreader/sumatrapdf#4673] [General]
Download address [Radio Regulations](https://www.itu.int/en/publications/ITU-R/pages/publications.aspx?parent=R-REG-RR-2024&media=electronic), I chose the Simplified Chinese version.

**Comment by kjk** [comment]
Can't reproduce with latest [pre-release](https://www.sumatrapdfreader.org/prerelease) version
